liiatools/datasets/s903/lds_ssda903_clean/populate.py

```diff
@@ -49,7 +49,10 @@ def create_la_child_id(event, la_code):
     elif isinstance(event.cell, float):
         child_id = int(event.cell)
     else:
-        child_id = event.cell
+        try:
+            child_id = event.cell.strip()
```

Comment:
A suggestion (not at all a requirement): 

it seems that that logic here is "strip white spaces if it's a string otherwise don't do anything". In that case, why not write the code to say just that? So:

```python
if isinstance(event.cell, str):
    child_id = event.cell.strip()
else:
    child_id = event.cell
```

Of course, both of them work and this doesn't really matter. But maybe it's more clear this way? What do you think?